File: Java/PetShop.java
class PetShop{
    private String ID;
    private String name;
    private String kategori;
    private int harga;

    PetShop(String ID, String nama, String kategori, int harga){
        this.ID = ID;
        this.name = nama;
        this.kategori = kategori;
        this.harga = harga;
    }
    //  Setter dan Getter untuk atribut yang ada
    public void setId(String id){
        this.ID = id;
    }

    public String getID(){
        return this.ID;
    }

    public void setName(String name){
        this.name = name;
    }

    public String getName(){
        return this.name;
    }

    public void setKategori(String kategori){
        this.kategori = kategori;
    }

    public String getKategori(){
        return this.kategori;
    }

    public void setHarga(int harga){
        this.harga = harga;
    } 

    public int getHarga(){
        return this.harga;
    }

    public void display(){
        System.out.println("----------------------------");
        System.out.println("Id      :" + ID);
        System.out.println("Name    :" + name);
        System.out.println("Kategori:" + kategori);
        System.out.println("Harga   :" + harga);
        System.out.println("----------------------------");
    }

    public void change(String ID, String name, String kategori, int harga){
        this.ID = ID;
        this.name = name;
        this.kategori = kategori;
        this.harga = harga;
    }
}

File: Python/PetShop.py
class PetShop:
    def __init__(self, ID, name, kategori, harga):
        self.__ID = ID
        self.__name = name
        self.__kategori = kategori
        self.__harga = harga
    
    # Getter dan setter untuk atribut yang ada
    def getID(self):
        return self.__ID
    def setID(self, ID):
        self.__ID = ID
    
    def getName(self):
        return self.__name
    def setName(self, name):
        self.__name = name
    
    def getKategori(self):
        return self.__kategori
    def setKategori(self, kategori):
        self.__kategori = kategori

    def getHarga(self):
        return self.__harga
    def setHarga(self, harga):
        self.__harga = harga

    def display(self):
        print("------------------------")
        print(f"Id      : {self.__ID}")
        print(f"Name    : {self.__name}")
        print(f"Kategori: {self.__kategori}")
        print(f"Harga   : {self.__harga}")
        print("------------------------")
    
    def change(self, Id, name, kategori, harga):
        self.__ID = Id
        self.__name = name
        self.__kategori = kategori
        self.__harga = harga